modules/sno-du-removing-the-console-operator.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-removing-the-console-operator_{context}"]
+= Disabling the console Operator
+
+The console-operator installs and maintains the web console on a cluster. When the node is centrally managed the Operator is not needed and makes space for application workloads.
```

Comment:
s/console-operator/Console Operator ??

How do I know if the node is centrally managed and I can safely delete the Operator?

Reply:
@imiller0 Clarification?

Reply:
The assumption is that all far-edge clusters running DU applications will be centrally managed and not require local console access.